tests/models/sho/vmc_sho.xml

```diff
@@ -34,7 +34,7 @@
     </wavefunction>
 
     <hamiltonian name="h0" type="generic" target="e">
-      <extpot type="HarmonicExt" mass="5.0" energy="0.3"/>
+      <extpot type="HarmonicExt" mass="1.0" energy="1.0"/>
```

Comment:
The input file of a test is changed but there is no change of the testing results.
I guess it was for development and should be reverted.
A statistic test should not hard code random seed.

Reply:
How should I proceed here?

Reply:
More of clarification needed here. Please explain
1. Why do you need to change parameters in an existing test? How wrong was the old value?
2. Why such changes doesn't impact the reference value?

Reply:
The external potential generated in Python for the HDF5 format file uses parameters mass = 1.0 and energy = 1.0. I can change the parameters in the Python script such that it matches the old testing parameters.

Reply:
@Paul-St-Young since you contributed the test, could you advice which is the way to go?

Reply:
Your change in the input file so far breaks the existing test short-sho-vmc-1-16. This needs to be addressed.